jovo-integrations/jovo-plugin-pulselabs/src/PulseLabs.ts

```diff
@@ -1,21 +1,23 @@
 import { BaseApp, HandleRequest, Plugin, PluginConfig } from 'jovo-core';
-import _merge = require('lodash.merge');
```

Comment:
same here. `lodash.merge` needs to be used to merge the configs from the constructor and `src/config.js` in the Jovo project folder